exercises/practice/two-bucket/.meta/example.ex

```diff
@@ -0,0 +1,55 @@
+defmodule TwoBucket do
+  defstruct [:goal, :moves, :other_bucket]
+  @type t :: %TwoBucket{goal: :one | :two, moves: integer, other_bucket: integer}
```

Comment:
Is this the clearest interface?

Would it not be clearer for a student studying the problem to retun their answer using a struct `%TwoBucker{bucket_one: x, bucket_two: y, moves: z}`

Reply:
It's the interface suggested in the tests. But you are right it's not super intuitive, I'll change it. Thank you!

Reply:
Cool, I think things like this are good reasons to break continuity with the problem specs repo

Reply:
Agreed! It reads much better now